peachjam_ml/apps.py

```diff
@@ -4,11 +4,3 @@
 class PeachjamMLConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "peachjam_ml"
-
-    def ready(self):
-        import peachjam_ml.signals  # noqa
-        from peachjam.views import BaseDocumentDetailView
-
-        from .handler import modify_document_detail_context
-
-        BaseDocumentDetailView.modify_context.connect(modify_document_detail_context)
```

Comment:
@Sandravaphilips we don't want to remove the permissions entirely, we only want to check if the user is logged in to allow them to view similar documents